src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/formats/html/AbstractMemberWriter.java

```diff
@@ -225,24 +220,19 @@ protected void addModifierAndType(Element member, TypeMirror type,
                 } else {
                     code.add(Entity.NO_BREAK_SPACE);
                 }
-                code.add(
-                        writer.getLink(new LinkInfoImpl(configuration,
-                        LinkInfoImpl.Kind.SUMMARY_RETURN_TYPE, type)));
-            } else {
-                code.add(
-                        writer.getLink(new LinkInfoImpl(configuration,
-                        LinkInfoImpl.Kind.SUMMARY_RETURN_TYPE, type)));
             }
-
+            code.add(
+                    writer.getLink(new LinkInfoImpl(configuration,
+                            LinkInfoImpl.Kind.SUMMARY_RETURN_TYPE, type)));
         }
         tdSummaryType.add(code);
     }
 
     /**
-     * Add the modifier for the member.
+     * Adds the modifier for the member.
      *
      * @param member the member to add the type for
-     * @param code the content tree to which the modified will be added
+     * @param code   the content tree to which the modified will be added
```

Comment:
I think this should be modifier not modified.

Reply:
Indeed; will fix